src/app/components/ItemPage/ItemPage.jsx

```diff
@@ -74,23 +74,50 @@ class ItemPage extends React.Component {
             <ul>
               {record[f.field].map((value, i) => {
                 const v = f.field === 'idOwi' ? value.substring(8) : value;
-                return <li key={i}><a target="_blank" href={f.url(v)}>{v}</a></li>;
+                return <li key={i}><a target="_blank" title={v} href={f.url(v)}>{v}</a></li>;
               })}
             </ul>
           ),
         });
 
       // list of links
       } else if (fieldValue['@id']) {
-        displayFields.push({ term: f.label, definition: <ul>{record[f.field].map((obj, i) => (<li key={i}><a onClick={(e) => this.onClick(e, `${f.field}:"${obj['@id']}"`)} href={`/search?q=${encodeURIComponent(`${f.field}:"${obj['@id']}"`)}`}>{obj.prefLabel}</a></li>))}</ul> });
+        displayFields.push(
+          { term: f.label,
+            definition: <ul>{record[f.field].map((obj, i) => (
+              <li key={i}>
+                <a
+                  onClick={e => this.onClick(e, `${f.field}:"${obj['@id']}"`)}
+                  title={`${f.field}:${obj['@id']}`}
+                  href={`/search?q=${encodeURIComponent(`${f.field}:"${obj['@id']}"`)}`}
+                >{obj.prefLabel}</a>
+              </li>))}
+            </ul>,
+          },
+        );
 
       // list of links
       } else if (f.linkable) {
-        displayFields.push({ term: f.label, definition: <ul>{record[f.field].map((value, i) => (<li key={i}><a onClick={(e) => this.onClick(e, `${f.field}:"${value}"`)} href={`/search?q=${encodeURIComponent(`${f.field}:"${value}"`)}`}>{value}</a></li>))}</ul> });
+        displayFields.push(
+          { term: f.label,
+            definition: <ul>{record[f.field].map((value, i) => (
+              <li key={i}>
+                <a
+                  onClick={e => this.onClick(e, `${f.field}:"${value}"`)}
+                  title={`${f.field}:"${value}"`}
```

Comment:
The value are correct, but each title needs a "Make a new search for " preceding it.